object-manipulation-1/src/set-value.js

```diff
@@ -0,0 +1,8 @@
+/* exported setValue */
+// Define a function named setValue that takes three arguments, object, key and value.
+//    Declare a variable named 'keyValue', and asisgn it to the result of the expression object at key = value.
+//    Return the value of the keyValue variable.
+function setValue(object, key, value) {
+  var keyValue = object[key] = value;
+  return keyValue;
+}
```

Comment:
This works but what we really want is to just setting the value to object with key:
```
function setValue(object, key, value) {
  object[key] = value;
}
```